File: src/main/webapp/js/service/blogService.js
app.service("blogService",function($http){
	//查询博客列表
	this.findBlogList = function(page,rows){
		return $http.get("./blog1/getBlogList.do?page="+page+"&rows="+rows);
	}
	
	//根据id查询博客详细内容
	this.findBlogById = function(blogId){
		return $http.get("./blog1/getBlogById.do?blogId="+blogId);
	}
	
	this.findBlogListByTypeId = function(page,rows,typeId){
		return $http.get("./blog1/getBlogListByTypeId.do?typeId="+typeId+"&page="+page+"&rows="+rows);
	}
});

File: src/main/java/com/blog/service/BlogService.java
package com.blog.service;


import com.blog.entity.PageResult;
import com.blog.pojo.TbBlog;

public interface BlogService {
	/**
	 * 根据博客id查询
	 * @param id
	 * @return
	 */
	TbBlog getBlogById(Integer id);
	/**
	 * 获取博客列表
	 * @return
	 */
	PageResult getBlogList(int page,int rows);
	
	/**
	 * 根据类别查询博客列表
	 * @param id
	 * @return
	 */
	PageResult getBlogListByTypeId(int page,int rows,Integer id);
}
 